ui/src/test/java/org/pentaho/di/ui/trans/steps/tableoutput/TableOutputDialogTest.java

```diff
@@ -66,4 +79,40 @@ private RowMetaInterface createRowMeta( String[] args, boolean hasEmptyFields )
     }
     return result;
   }
+
+  private void isConnectionSupportedTest( boolean supported ) {
+
+    TableOutputDialog dialog = mock( TableOutputDialog.class );
+
+    TransMeta transMeta = mock( TransMeta.class );
+    DatabaseMeta dbMeta = mock( DatabaseMeta.class );
+    TextVar text = mock( TextVar.class );
+    CCombo combo = mock( CCombo.class );
+    DatabaseInterface dbInterface = mock( DatabaseInterface.class );
+
+    Whitebox.setInternalState( dialog, "wTable", text );

```

Comment:
Import the static method instead of the `Whitebox` class just like you're doing with all the `when`, `doNothing`, `verify`, _etc_... methods.